CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ This CHANGELOG follows the format listed at [Keep A Changelog](http://keepachang
 
 ## [Unreleased]
 
+## [8.4.0] - 2017-09-29
```

Comment:
Please leave the version bumping and dating to the maintainers. All changes should stay under `## [Unreleased]` until a maintainer is ready to create a release. https://github.com/sensu-plugins/community/blob/master/HOW_WE_CHANGELOG.md#unreleased-change